code/controllers/subsystems/zcopy.dm

```diff
@@ -198,7 +198,8 @@ SUBSYSTEM_DEF(zcopy)
 	calculate_zstack_limits()
 
 	for (var/zlev in 1 to world.maxz)
-		for (var/turf/T in block(locate(1, 1, zlev), locate(world.maxx, world.maxy, zlev)))
+		var/datum/level_data/level = SSmapping.levels_by_z[zlev]
+		for (var/turf/T as anything in block(level.level_inner_min_x, level.level_inner_min_y, zlev, level.level_inner_max_x, level.level_inner_max_y))
```

Comment:
Is it impossible for turfs to be placed outside of this? This is supposed to be a universal reset, though it's explicitly unsupported anyways.

Reply:
technically turfs outside of it exist, but should not participate in sim _at all_. it is currently invalid for unsim filler turfs outside level bounds to have any sort of behavior especially ZM behavior. i guess at some point if we get a virtual-Z system a la shiptest it could be, but we'd probably still have a level data datum associated with it even then